source/assets/scripts/api.js

```diff
@@ -163,3 +163,18 @@ function remove_custom(id){
     }
   }
 }
+
+/**
+ * This function will get a array of intolerance then store them into local storage with the key intolerance
+ * @param {array of strin } intolerance_list  a list of intolerance that user applied
```

Comment:
Try this: (I'm not sure)
` * @param {array} intolerance_list - a list of intolerance that user applied`

Reply:
Generate Doc / deploy test passed